pom.xml

```diff
@@ -82,6 +82,13 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>2.5.3</version>
+        <inherited>true</inherited>
```

Comment:
As per the documentation inherited property allows plugin configuration propagation to child poms.
See - https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_inherited_Tag_In_Build_Plugins

Regarding .properties file - not much configuration is needed here I think.